validator/validator.py

```diff
@@ -341,7 +342,7 @@ def fetch_schema(schema_url):
         r = requests.get(schema_url)
         r.raise_for_status()
         schema = r.text
-        return anymarkup.parse(schema, force_types=None)
+        return parse_anymarkup(schema)
```

Comment:
I fail to understand how this is actually working, since you're passing a yaml or json string and `parse_anymarkup` expects a file...

Reply:
Nice find. Indeed the `anymarkup` package has detection of format for text/bytes objects and will parse the markup accordingly.

This is for when the schema definition points to a URL. In all cases (for now) they are json documents. Since we know we will always get json here (for now anyway), I have changed this to `json.load()`

Reply:
It makes sense to assume json there

Reply:
fixed. from the looks of it that code path is never taken with what we currently have in app-interface